packages/blazor-workspace/Tests/NimbleBlazor.Tests.Acceptance/SharedPlaywrightCollectionDefinition.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Xunit;
+﻿using BlazorWorkspace.Testing.Acceptance;
+using Xunit;
```

Comment:
I guess I'm curious as to why this class couldn't move to the shared project.

Reply:
If I move it to the shared project, we get the build error: `xUnit1041: Fixture argument 'playwrightFixture' does not have a fixture source (if it comes from a collection definition, ensure the definition is in the same assembly as the test)`.